hash-tables/ex1/ex1.py

```diff
@@ -13,7 +13,24 @@ def get_indices_of_item_weights(weights, length, limit):
     YOUR CODE HERE
     """
 
-    return None
+    if length == 1:
+        result = None
+
+    for i in range(length):
+        hash_table_insert(ht, weights[i], i)
+
+    for i in range(length):
+        package_1 = weights[i]
+        package_2 = limit - package_1
+        search = hash_table_retrieve(ht, package_2)
+
+        if search is not None:
+            if search >= i:
```

Comment:
Take a look at this link here [list sort](https://docs.python.org/2/howto/sorting.html#ascending-and-descending) for a way to condense your `if/elif` block and return your `result` in proper order with a `one-liner` 👍